README.md

```diff
@@ -150,3 +150,18 @@ Yes, you can configure notification in the Crawler UI here: <https://crawler.alg
 - Need help? We have you covered in our [Discourse forum](https://discourse.algolia.com/c/netlify/28)
 - Found a bug in the plugin? Please read our [Contributing guide](/CONTRIBUTING.md) and post an issue in this repository
 - Did not find any answer? Please reach out to `support@algolia.com`
+
+## Architecture
```

Comment:
Not sure it's useful for 99% of the readers

Reply:
Well if you're on the repo, there a non-negligible chance you're here to explore the code.
It's the last section of the README, why would that be an issue?

Reply:
it's not an issue it's just we are redirecting all the people here for the moment, 
so most of the people will not be interested in the code

Reply:
Which is fine as this will be the last section.